releases/cosmwasm/2025-02-xrplevm-GMP-v6.4.0.md

```diff
@@ -0,0 +1,351 @@
+# You should see something like this:
+{
+  "data": {
+    "name": "xrplevm",
+    "gateway": {
+      "address": "axelar1hzz0s0ucrhdp6tue2lxk3c03nj6f60qy463we7lgx0wudd72ctmsee8enx"
+    },
+    "frozen_status": 0,
+    "msg_id_format": "hex_tx_hash_and_event_index"
+  }
+}
+```
+
+6. Update ampd with the `xrplevm` chain configuration. Verifiers should use their own `xrplevm` RPC node for the `http_url` in production.
+
+| Network              | `http_url`              |
+| -------------------- | ----------------------- |
+| **Devnet-amplifier** | https://rpc.xrplevm.org |
+| **Stagenet**         | https://rpc.xrplevm.org |
+| **Testnet**          | https://rpc.xrplevm.org |
+| **Mainnet**          | ``                      |
+
+```bash
+[[handlers]]
+chain_finalization="RPCFinalizedBlock"
+chain_name="xrplevm"
+chain_rpc_url=[http url]
+cosmwasm_contract="$VOTING_VERIFIER"
+type="EvmMsgVerifier"
+
+[[handlers]]
+chain_finalization="RPCFinalizedBlock"
+chain_name="xrplevm"
+chain_rpc_url=[http url]
+cosmwasm_contract="$VOTING_VERIFIER"
+type="EvmVerifierSetVerifier"
+```
+
+7. Update ampd with the `xrplevm` chain configuration.
+
+```bash
+ampd register-chain-support "[service name]" xrplevm
+```
+
+8. Register prover contract on coordinator
+
+```bash
+node cosmwasm/submit-proposal.js execute \
+  -c Coordinator \
+  -t "Register Multisig Prover for xrplevm" \
+  -d "Register Multisig Prover address for xrplevm at Coordinator contract" \
+  --runAs $RUN_AS_ACCOUNT \
+  --deposit $DEPOSIT_VALUE \
+  --msg "{
+    \"register_prover_contract\": {
+      \"chain_name\": "xrplevm",
+      \"new_prover_addr\": \"$MULTISIG_PROVER\"
+    }
+  }"
+```
+
+9. Authorize `xrplevm` Multisig prover on Multisig
+
+```bash
+node cosmwasm/submit-proposal.js execute \
+  -c Multisig \
+  -t "Authorize Multisig Prover for xrplevm" \
+  -d "Authorize Multisig Prover address for xrplevm at Multisig contract" \
+  --runAs $RUN_AS_ACCOUNT \
+  --deposit $DEPOSIT_VALUE \
+  --msg "{
+    \"authorize_callers\": {
+      \"contracts\": {
+        \"$MULTISIG_PROVER\": "xrplevm"
+      }
+    }
+  }"
+```
+
+```bash
+axelard q wasm contract-state smart <multisig-addr> '{"is_caller_authorized": {"contract_address": "<xrplevm-multisig-prover-addr>", "chain_name": "xrplevm"}}' --output json | jq .
+# Result should look like:
+{
+  "data": true
+}
+```
+
+10. Create reward pool for voting verifier
+
+#### Rewards
+
+| Network              | `epoch_duration` | `participation_threshold` | `rewards_per_epoch` |
+| -------------------- | ---------------- | ------------------------- | ------------------- |
+| **Devnet-amplifier** | `100`            | `["7", "10"]`             | `100`               |
+| **Stagenet**         | `600`            | `["7", "10"]`             | `100`               |
+| **Testnet**          | `14845`          | `["7", "10"]`             | `100`               |
```

Comment:
@talalashraf do you think we should use a smaller value than `14845` for testnet for testing purposes?